src/main/groovy/com/bmuschko/gradle/docker/tasks/image/DockerCommitImage.groovy

```diff
@@ -23,18 +23,11 @@ import org.gradle.api.tasks.Internal
 import org.gradle.api.tasks.Optional
 
 class DockerCommitImage extends DockerExistingContainer {
-    /**
-     * Repository.
-     */
-    @Input
-    @Optional
-    final Property<String> repository = project.objects.property(String)
```

Comment:
If we're removing this property a "better" name might be to just use `image` as that technically encompasses the entire URL. It's at least what is used within the docker docs all over the place.

Reply:
done